Java-FlightServer/client.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd build/classes/java/main
+java --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED -Dlogback.configurationFile=file:../../../../logging.xml -cp "../../../libs/Java-FlightServer-1.0-SNAPSHOT-all.jar:." ibm.com.example.MyFlightClient
```

Comment:
1. to disable logging messages:
**-Dlogback.configurationFile=file:../../../../logging.xml**

2. to remove warning:
WARNING: Illegal reflective access by org.apache.arrow.memory.util.MemoryUtil
to field java.nio.Buffer.address
**--add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED**

Reply:
Did you try adding `logback.xml` to the `resources` folder? blind guess:
```xml
<?xml version="1.0" encoding="UTF-8" ?>
<configuration>
  <statusListener class="ch.qos.logback.core.status.NopStatusListener"/>
  <logger name="org.apache.arrow" level="OFF"/>
</configuration>
```

Reply:
@roee88: logback.xml did not work for me